docs/pipeline/eager/crick.md

```diff
@@ -0,0 +1,30 @@
+# nf-core/configs: Crick eager specific configuration
+
+Extra specific configuration for eager pipeline
+
+## Pipeline specific profiles
+
+Specific-pipeline profiles exist for the following pipelines, and will be automatically loaded when supplying `-profile crick` to your command:
+
+- eager
+
+## Usage
+
+To use, run the pipeline with `-profile crick`.
+
+This will download and launch the eager specific [`crick.config`](../../../conf/pipeline/eager/crick.config) which is adapted to data generated at the Crick and
+processed on the Crick's HPC.
+
+Example: `nextflow run nf-core/eager -profile crick`
+
+### screening
+
+- Low coverage data.
+
+### production
+
+- High coverage data.
+
```

Comment:
Might be good to specify what is the difference is between the two (or a rough guide) - I guess it depends on whether 'screenin/production' is official terms in the aDNA group or not or just a rule-of-thumb?